go/ql/test/library-tests/semmle/go/frameworks/SQL/QueryString.ql

```diff
@@ -8,10 +8,10 @@ class SqlTest extends InlineExpectationsTest {
 
   override predicate hasActualResult(Location location, string element, string tag, string value) {
     tag = "query" and
-    exists(SQL::Query q, SQL::QueryString qs, int qsLine | qs = q.getAQueryString() |
+    exists(SQL::Query q, SQL::QueryString qs | qs = q.getAQueryString() |
       q.hasLocationInfo(location.getFile().getAbsolutePath(), location.getStartLine(),
         location.getStartColumn(), location.getEndLine(), location.getEndColumn()) and
-      qs.hasLocationInfo(_, qsLine, _, _, _) and
+      qs.hasLocationInfo(_, _, _, _, _) and
```

Comment:
Maybe worth a quick dive in git to figure out what this constraint was supposed to mean

Reply:
Good point. Since this part of the test was first added, this line has been there asserting nothing more than "`qs` has a location". I don't think that needs to be asserted. I don't think the intention was that the location should be linked to the location of `q`. So I've removed this line now.